.jenkins/common_job_properties.groovy

```diff
@@ -155,6 +164,20 @@ class common_job_properties {
     setPullRequestBuildTrigger(context, comment)
   }
 
+  // Enable triggering postcommit runs against pull requests. Users can comment the trigger phrase
+  // specified in the postcommit job and have the job run against their PR to run
+  // tests not in the presubmit suite for additional confidence.
+  static def enableTriggeringOnPreCommit(def context,
+                                         def check_title,
```

Comment:
commitStatusContext feels really opaque to me -- context is a domain specific term to the groovy DSL which means very little to anyone inexperienced, and it's not really the commit status, it's the title of the check running in the PR. I can change to pr_check_title or something if you want?